cardano-cli/test/cardano-cli-golden/files/golden/governance/stakeaddress/alwaysAbstainDeleg.cert

```diff
@@ -0,0 +1,5 @@
+{
+    "type": "CertificateShelley",
+    "description": "Stake Address Delegation Certificate",
```

Comment:
Not for this PR but should be update the description to read something like "Stake Address Vote Delegation Certificate"?

Reply:
Addressed on the latest commit. I recovered the names we had when the command was under Dreps.hs